cluster/gce/manifests/kube-apiserver.manifest

```diff
@@ -167,6 +167,11 @@
   {% set enable_garbage_collector = "--enable-garbage-collector=" + pillar['enable_garbage_collector'] -%}
 {% endif -%}
 
+{% set etcd_compaction_interval = "" %}
```

Comment:
I'd use the `_sec` suffix, since you're appending "s" to the value.

Reply:
Changed partially (to make it compatible with there is for kube_apiserver_request_timeout_sec).
BTW - it seems that we no longer have proper support for those pillars, so we should probably get rid of them completely...